OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -35,10 +35,10 @@ public class MissileInfo : IProjectileInfo
 		public readonly bool Shadow = false;
 
 		[Desc("Minimum vertical launch angle (pitch).")]
-		public readonly WAngle MinimumLaunchAngle = WAngle.Zero;
+		public readonly WAngle MinimumLaunchAngle = new WAngle(-64);
 
 		[Desc("Maximum vertical launch angle (pitch).")]
-		public readonly WAngle MaximumLaunchAngle = new WAngle(64);
+		public readonly WAngle MaximumLaunchAngle = new WAngle(128);
```

Comment:
Do we want to enable them to seemingly shoot straight up, though? (Or down, for that matter)
The artwork doesn't suggest it should.